client/src/pages/Signup.js

```diff
@@ -58,11 +68,30 @@ class SignUpPage extends Component {
 
   handleSubmit = event => {
     event.preventDefault();
```

Comment:
The validators will need to be called here also 

handleSubmit = event => {
    event.preventDefault();
    const emailIsValid = this.emailValidate();
    const nameIsValid = this.nameValidate();
    const passwordIsValid = this.passwordValidate();
    if (emailIsValid && nameIsValid && passwordIsValid) {
      console.log(this.state);
      this.setState(initalState);
    }
  };

Let me know what you think!

Reply:
I'm struggling to display password validation error